packages/ui-library/src/components/textarea/index.stories.ts

```diff
@@ -319,34 +319,34 @@ export default {
     },
 
     // Events
-    onChange: {
-      name: 'onChange',
+    blrChange: {
+      name: 'blrChange',
       description: 'Fires when the value changes.',
-      action: 'onChange',
+      action: 'blrChange',
       table: {
         category: 'Events',
       },
     },
-    onFocus: {
-      name: 'onFocus',
-      description: 'Fires when the component is focused.',
-      action: 'onFocus',
+    blrSelect: {
+      name: 'blrSelect',
```

Comment:
name overwrite the origin property name and show us in storybook what we want to see (instead of the origin).  I used it sometimes to show sizeVariant in the documentation correctly. In that case we don't need this line because the both have the same value."name"  is no longer needed and can be removed.